Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1782,28 +1798,23 @@ void CGContextDrawGlyphRun(CGContextRef context, const DWRITE_GLYPH_RUN* glyphRu
 }
 #pragma endregion
 
-CGImageRef CGPNGImageCreateFromFile(NSString* path) {
-    return new CGPNGDecoderImage([path UTF8String]);
-}
-
-CGImageRef CGPNGImageCreateFromData(NSData* data) {
-    return new CGPNGDecoderImage(data);
-}
-
-CGImageRef CGJPEGImageCreateFromFile(NSString* path) {
-    return new CGJPEGDecoderImage([path UTF8String]);
-}
-
-CGImageRef CGJPEGImageCreateFromData(NSData* data) {
-    return new CGJPEGDecoderImage(data);
-}
-
 #pragma region CGBitmapContext
 struct __CGBitmapContextImpl {
     woc::unique_cf<CGImageRef> image;
 };
 
-struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGBitmapContextImpl, __CGContext> {};
+struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGBitmapContextImpl, __CGContext> {
+    inline ComPtr<ID2D1RenderTarget>& RenderTarget() {
```

Comment:
Talked offline. Moving the creation into Quartz

---
In reply to: [87046467](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87046467) [](ancestors = 87046467)